docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -37,7 +37,12 @@ return [
 ];
 ```
 
-The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration.
+The factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` uses the configuration, searches for the configuration key `caches` and creates the storage adapters using the discovered configuration. 
+
+> ### Cache named `config` is not possible
+>
+> A cache named `config` is not possible due to internal service conflicts with MVC configuration.
+> The service named `config` is reserved for he projects configuration and thus cannot be used with the `caches` configuration.
```

Comment:
Admonitions/callouts should be used:

```markdown
WARNING: **Cache Named `config` Is Not Possible**
A cache named `config` is not possible due to internal service conflicts with MVC configuration.
The service named `config` is reserved for he projects configuration and thus cannot be used with the `caches` configuration.
```

https://oprypin.github.io/markdown-callouts/index.html#usage

Reply:
Markdown is unsuitable for technical documentations, the syntax is not sufficient and extensions are crap. :stuck_out_tongue_winking_eye: